composer/lib/dependabot/composer/helpers.rb

```diff
@@ -16,13 +16,13 @@ module Helpers
       def self.composer_version(composer_json, parsed_lockfile = nil)
         if parsed_lockfile && parsed_lockfile["plugin-api-version"]
```

Comment:
Should we use `.fetch` here to protect against a key error if there is a missing `plugin-api-version`?

Reply:
I had to look this one up as I didn't know 😁 

For a missing key:
* `.fetch` raises a `KeyError`, which we'd need to catch...
* `[key]` returns `nil`, which is "falsey", so a missing key will result in this `if` condition failing.

So IIUC, we want it to stay like it is actually...